spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -6,15 +6,67 @@
 package com.datastrato.gravitino.spark.connector.hive;
 
 import com.datastrato.gravitino.spark.connector.PropertiesConverter;
+import com.google.common.collect.ImmutableMap;
 import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
+import java.util.Optional;
+import javax.ws.rs.NotSupportedException;
 
-/** Transform hive catalog properties between Spark and Gravitino. Will implement in another PR. */
+/** Transform hive catalog properties between Spark and Gravitino. */
 public class HivePropertiesConverter implements PropertiesConverter {
```

Comment:
Why do you introduce a new `PropertiesConverter` in Spark, does `com.datastrato.gravitino.catalog.property.PropertyConverter` satisfy your need?

Reply:
PropertyConverter is mainly used to convert from `a` to `b` and back `b` to `a`,  the property transform logic in  Spark is more complicated